code/defines/procs/announce.dm

```diff
@@ -1,50 +1,48 @@
 GLOBAL_DATUM_INIT(minor_announcement, /datum/announcement/minor, new())
 GLOBAL_DATUM_INIT(priority_announcement, /datum/announcement/priority, new(do_log = 0))
-GLOBAL_DATUM_INIT(command_announcement, /datum/announcement/priority/command, new(do_log = 0, do_newscast = 0))
-GLOBAL_DATUM_INIT(event_announcement, /datum/announcement/priority/command/event, new(do_log = 0, do_newscast = 0))
+GLOBAL_DATUM_INIT(command_announcement, /datum/announcement/priority/command, new(do_log = 0))
+GLOBAL_DATUM_INIT(event_announcement, /datum/announcement/priority/command/event, new(do_log = 0))
```

Comment:
Could you make these TRUE/FALSE while youre at it please?

Reply:
Should be good now.